command/resource/helper.go

```diff
@@ -35,3 +37,20 @@ func ParseInputParams(inputArgs []string, flags *flag.FlagSet) error {
 	}
 	return nil
 }
+
+func GetTypeAndResourceName(args []string) (gvk *api.GVK, resourceName string, e error) {
+	// it has to be resource name after the type
+	if strings.HasPrefix(args[1], "-") {
+		return nil, "", fmt.Errorf("Must provide resource name right after type")
+	}
+
+	s := strings.Split(args[0], ".")
+	gvk = &api.GVK{
+		Group:   s[0],
+		Version: s[1],
+		Kind:    s[2],
+	}
```

Comment:
This will cause a panic if the resource type is not in `a.b.c` format. 
We need a check here to prevent that
```
if len(s) < 3 {
		return nil, "", fmt.Errorf("Must include resource type argument in group.verion.kind format")
	}
```

Also, lets add a test case for it

Reply:
updated, and added one more test case for it.
Notice I restrict the condition to 
```
if len(s) != 3 {}
```